lexicons/com/atproto/data/uploadBlob.json

```
@@ -0,0 +1,32 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.data.uploadBlob",
+  "type": "procedure",
+  "description": "Upload a blob to later be referenced in user repo",
+  "input": {
+    "encoding": "multipart/form-data",
+    "schema": {
+      "type": "object",
+      "required": ["did",  "data"],
+      "properties": {
+        "did": {
+          "type": "string",
+          "description": "The DID of the repo."
+        },
+        "data": {
+          "type:": "blob"
```

This popped out at me, and might be hard to spot:
```suggestion
          "type": "blob"
```

---

lmao dude i actually have this fix locally. took a lil bit of debugging to find that one 😅